changelog/unreleased/issue-1570

```diff
@@ -0,0 +1,5 @@
+Enhancement: Multiple hostnames for `dump`, `find`, `forget`, `ls`, `mount`, `restore`, `snapshots`, `stats`, `tag`.
+
+Added the possibility to specify multiple `--hostname` or `-H` flags where it makes sense.
```

Comment:
The title line can't end with a dot, it shouldn't have any punctuation. I also suggest shortening it, please consider changing these three lines to the following (note that I also changed `--hostname` to `--host` since the former is deprecated):

~~~
Enhancement: Support specifying multiple host flags for various commands

Previously commands didn't take more than one `--host` or `-H` argument into account, which could be limiting with e.g. the `forget` command.

The `dump`, `find`, `forget`, `ls`, `mount`, `restore`, `snapshots`, `stats` and `tag` commands will now take into account multiple `--host` and `-H` flags.
~~~